static/app/utils/metrics/index.tsx

```diff
@@ -223,6 +231,51 @@ export function getReadableMetricType(type?: string) {
   return metricTypeToReadable[type as MetricType] ?? t('unknown');
 }
 
+const MILLISECOND = 1;
+const MICROSECOND = MILLISECOND / 1000;
+
+export function formatDuration(seconds: number): string {
+  if (!seconds) {
+    return '0ms';
```

Comment:
The current implementation would also fall back to `ms`. Ideally the formatter would be aware of the chart's series and use the same unit here as for the second smallest axis value. 🤔